### DISCUSSION TITLE
New cPanel Install v2.36.3

### DISCUSSION THREAD

**Discussion by HighlyFavoredBA** [the-djmaze/snappymail#1636] [Q&A]
I extracted the ZIP for v2.36.3, and I have that in public_html/snap for testing.  Site.com/snap opens a page that only says SnappyMail in the top left corner.  How do I actually install the software, Without using my WHM account?

How can I add custom animated emojis via a emoji plugin?!  And how can I add a global signature?!  :--)

**Comment by the-djmaze** [comment]
Which zip?
There is the normal SnappyMail, and a specific cPanel version to insttall as root/admin.

Animations (gif) are not supported due to Outlook.
So what do you mean by emoji's?

Signatures are by identity, not global

**Reply by HighlyFavoredBA** [reply]
If you know how to add global signatures, that would be excellent!  😎

Emoji's / Emoticons (Smileys).  Non animated emotions to me are boring!   😎

I'm Away from my computer.  I Don't recall what zip I installed and tested.  Please post the correct zip link in this thread.  😎

I've long used Roundcube due to the ability to add global signatures, the ability to add custom emoticons, etc !  In recent years, TinyMCE upgraded their emoticons plugin in a way that Isn't easy to add custom emoticons.  🙃

**Reply by the-djmaze** [reply]
see https://github.com/the-djmaze/snappymail/wiki/Installation-instructions#cpanel

**Reply by the-djmaze** [reply]
> Emoji's / Emoticons (Smileys). Non animated emotions to me are boring! 😎

You probably love the Microsoft Teams Fluent emoji's 🙈

**Comment by HighlyFavoredBA** [comment]
😀
